README.md

```diff
@@ -113,13 +117,13 @@ The `Value` representation is sufficient for very basic tasks but can be tedious
 to work with for anything more significant. Error handling is verbose to
 implement correctly, for example imagine trying to detect the presence of
 unrecognized fields in the input data. The compiler is powerless to help you
-when you make a mistake, for example imagine typoing `v["name"]` as `v["nmae"]`
+when you make a mistake, for example imagine typing `v["name"]` as `v["nmae"]`
 in one of the dozens of places it is used in your code.
 
 ## Parsing JSON as strongly typed data structures
 
 Serde provides a powerful way of mapping JSON data into Rust data structures
-largely automatically.
+largely automatically. Also strongly typed data structure, insures that after you deserialize the json string and try to access a key of the object of type `String`, it will return the typed one and you will not have the problem of `""` when using the value, unlike the untyped example on top.
```

Comment:
Please use linebreaks instead of one long line.

* *a* strongly typed
* no comma before `insures`
* `insures` -> `ensures`
* it will return the typed one instead of a `Value`.
    * I don't think the `""` example is really useful here.